packages/gst_plugins_good.rb

```diff
@@ -4,53 +4,59 @@ class Gst_plugins_good < Package
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_good-1.18.4-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_good-1.18.4-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_good-1.18.4-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_good-1.18.4-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gst_plugins_good-1.18.4-1-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: 'e0fdc371c57de6ec9ce08478f3ed37d11c6c9b732266b2a07e53a5da38a4038e',
-     armv7l: 'e0fdc371c57de6ec9ce08478f3ed37d11c6c9b732266b2a07e53a5da38a4038e',
-       i686: '820d158973841c3e18ef1150b99586787f6fc95f4b18ce9c447a29da75270015',
-     x86_64: 'a4efb5f9552bcd9e6fcc5405f0c41209a9efc8c34404eb043d259d4cf2284cd5'
+    aarch64: 'c6bd8050a3c0f1b314c710a325211955179befb161ad265c6aeea8aec1e4f262',
+     armv7l: 'c6bd8050a3c0f1b314c710a325211955179befb161ad265c6aeea8aec1e4f262',
+       i686: '9d81667abfe65600e5b248de3c2de29ce6eb7f1e352929015c033809423324ea',
+     x86_64: '1ce70592cf499cda3a46f519db2ba479e0e8ca56acf43d44366b884d59b73487'
   })
 
-  depends_on 'aalib'
-  depends_on 'cairo'
-  depends_on 'gdk_pixbuf'
-  depends_on 'glib'
-  depends_on 'gst_plugins_base'
-  depends_on 'gstreamer'
-  depends_on 'gtk3'
-  depends_on 'jack'
-  depends_on 'libdv'
-  depends_on 'libgudev'
-  depends_on 'libjpeg'
-  depends_on 'libmp3lame'
-  depends_on 'libpng'
-  depends_on 'libsoup'
-  depends_on 'libsoup2'
-  depends_on 'libvpx'
-  depends_on 'libx11'
-  depends_on 'libxdamage'
-  depends_on 'libxext'
-  depends_on 'libxfixes'
+  # L = Logical Dependency, R = Runtime Dependency
+  depends_on 'libjpeg' => :build
   depends_on 'nasm' => :build
-  depends_on 'orc'
-  depends_on 'pipewire'
-  depends_on 'pulseaudio'
-  depends_on 'speex'
-  depends_on 'taglib'
-  depends_on 'v4l_utils'
-  depends_on 'wavpack'
+  depends_on 'gst_plugins_base' # L
+  depends_on 'aalib' # R
+  depends_on 'cairo' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'glib' # R
+  depends_on 'gst_plugins_base' # R
+  depends_on 'gstreamer' # R
+  depends_on 'gtk3' # R
+  depends_on 'jack' # R
+  depends_on 'libavc1394' # R
+  depends_on 'libcaca' # R
+  depends_on 'libdv' # R
+  depends_on 'libgudev' # R
+  depends_on 'libiec61883' # R
+  depends_on 'libmp3lame' # R
+  depends_on 'libpng' # R
+  depends_on 'libraw1394' # R
+  depends_on 'libsoup2' # R
+  depends_on 'libvpx' # R
+  depends_on 'libx11' # R
+  depends_on 'libxdamage' # R
+  depends_on 'libxext' # R
+  depends_on 'libxfixes' # R
+  depends_on 'mpg123' # R
+  depends_on 'orc' # R
+  depends_on 'pipewire' # R
+  depends_on 'pulseaudio' # R
+  depends_on 'speex' # R
+  depends_on 'taglib' # R
+  depends_on 'v4l_utils' # R
+  depends_on 'wavpack' # R
```

Comment:
This dependency list is still a bit longer than necessary.  gtk3 takes care of a lot of these implicitly.